docs/components/using-other-components.md

```diff
@@ -0,0 +1,90 @@
+# Using other components
+
+Once your component has been created you might have to "link" it to other component such as flare, status page, health,
+...
+
+This page document what needs to be done for each of them so you component is fully integrated in the Agent life cycle
+
+## Flare
+
+You need to migrate the code related to your component's domain from `pkg/flare` to your component. Once the migration
+of the Agent to components we should be able to delete `pkg/flare`.
+
+### Creating you callback
+
+In order to add data to a flare you need to register a provider.
+
+First create a `flare.go` file in your component (the file name is a convention) and create a `func (c *yourComp) fillFlare(fb flarehelpers.FlareBuilder) error` function.
+
+This function will be called everytime the agent generate a flare, either from the CLI or from the running agent. This
+callback will receive a `comp/core/flare/helpers:FlareBuilder`. The `FlareBuilder` is an interface that provides all the
+required helpers needed to add data to a flare (add files, copy directory, scrub data, ...).
+
+Example of `flare.go`:
+
+```golang
+import (
+	yaml "gopkg.in/yaml.v2"
+
+	flarehelpers "github.com/DataDog/datadog-agent/comp/core/flare/helpers"
```

Comment:
Maybe rename:
```suggestion
	flarebuilder "github.com/DataDog/datadog-agent/comp/core/flare/builder"
```

Reply:
This also contains the helper to declare a callback.